azure-stack/hci/concepts/azure-stack-hci-dashboard.md

```diff
@@ -0,0 +1,60 @@
+---
+title: Azure Stack HCI dashboard
+description: Learn to view and navigate your Azure Stack HCI dashboard in Azure portal.
+ms.topic: conceptual
+author: alkohli
+ms.subservice: azure-stack-hci
+ms.author: alkohli
+ms.reviewer: alkohli
+ms.date: 02/13/2024
+---
+
+# Azure Stack HCI dashboard
+
```

Comment:
Change to: Monitor Azure Stack HCI systems using the dashboard (preview)

Reply:
Done > Updated the H1.